RacingGame/Collision/src/main/java/racing/collision/CollisionSystem.java

```diff
@@ -0,0 +1,81 @@
+package racing.collision;
+
+import java.awt.geom.Area;
+import java.awt.Rectangle;
+import java.awt.Shape;
+import java.awt.geom.AffineTransform;
+import java.lang.Math;
+import racing.common.player.Player;
+import racing.commonnpc.NPC;
+import racing.common.map.Tile;
+import racing.common.data.Entity;
+import racing.common.data.GameData;
+import racing.common.data.GameImage;
+import racing.common.data.World;
+import racing.common.data.entityparts.MovingPart;
+import racing.common.data.entityparts.PositionPart;
+import racing.common.data.entityparts.TilePart;
+import racing.common.services.IPostEntityProcessingService;
+import java.util.UUID;
+
+
+public class CollisionSystem implements IPostEntityProcessingService {
+
+    @Override
+    public void process(GameData gameData, World world) {
+        Player player = (Player) world.getEntities(Player.class).get(0);
```

Comment:
Perhaps make this a for-loop as well, to have better support in case of future multiplayer partybus?

Reply:
Resolved in cd4797a